akka-stream/src/main/scala/akka/stream/stage/Stage.scala

```diff
@@ -341,8 +344,8 @@ abstract class StatefulStage[In, Out] extends PushPullStage[In, Out] {
    */
   abstract class State extends StageState[In, Out]
 
-  private var emitting = false
-  private var _current: StageState[In, Out] = _
+  private[this] var emitting = false
+  private[this] var _current: StageState[In, Out] = _
```

Comment:
I'm not sure about this. I know it's idomatic scala, but we have seen some surprises around it when it comes to binary compatibility evolution, and we "normally" don't use it anyway